### ISSUE TITLE
[Negative]2018-03-22

### ISSUE THREAD

**Issue by haoyang-udacity** [udacity/aind-issue-reports#236]
Was pretty boring\, no videos... The advantage of Tensorflow compared to simpler Keras was not at all obvious!
